template/ios/carthage/PROJECT.playground/Sources/NefPlaygroundSupport.swift

```diff
@@ -1,4 +1,4 @@
-import UIKit
+import SwiftUI
```

Comment:
Why do you need to import SwiftUI here? It creates an unnecessary dependency.

Reply:
In fact, it is not necessary, we can continue using UIKit